public_html/layouts/basic/modules/Settings/PDF/resources/Import.js

```diff
@@ -0,0 +1,17 @@
+/* {[The file is published on the basis of YetiForce Public License 3.0 that can be found in the following directory: licenses/LicenseEN.txt or yetiforce.com]} */
+'use strict';
+
+jQuery.Class('Settings_PDF_Import_Js', {}, {
+  /**
+   * Function to register event for PDF import
+   */
+  registerEvents: function() {
+    let form = $('.js-validation-engine');
+    form.validationEngine();
+  }
+});
+
+jQuery(document).ready(function() {
+	var settingPDFImportInstance = new Settings_PDF_Import_Js();
```

Comment:
use const/let instead of var when it is possible
```suggestion
	const settingPDFImportInstance = new Settings_PDF_Import_Js();
```

Reply:
done